src/components/MediaPlayer/MainControl/VolumeBar/VolumeBar.js

```diff
@@ -65,28 +58,48 @@ export class VolumeBarComponent {
     if (this.props.audio.muted) {
       this.volumeBar.value = this.currentVolume * 100;
       this.props.audio.muted = false;
-      this.volumeBarGradient(this.volumeBar.value);
     } else {
       this.props.audio.muted = true;
       this.volumeBar.value = 0;
-      this.volumeBarGradient(this.volumeBar.value);
     }
     this.toggleVolume();
   }
 
+  showModal() {
+    if (this.extraDiv) {
```

Comment:
maybe it would be better to just have this in template with display: none by default. And then set display: block when you need?